### DISCUSSION TITLE
Using GitHub for personal content projects

### DISCUSSION THREAD

**Discussion by YuriGor** [community/community#22551] [New to GitHub]
<p>Hi everyone, I am working on the presentational SAAS, kind of images layout editor. <br>
Users will create/update some presentations with images/text/other content.</p>
<p>I want to implement optional feature to allow users to store the history of modifications in their own github projects public or private.<br>
It will be one or couple of JSON files, images (or may be without images, if this is prohibited) and readme file with description of the presenatation and maybe screenshot of current state.</p>
<p>Each user will have to register his GitHub account if still has no one, and then grant an acces via API to my app.</p>
<p>So my question:<br>
does it fullfil the terms of githhub service?<br>
I mean is it allowed to have personal projects not related to any kind of IT development, but just arbitrary presentation document?</p>
<p>Latest public version of the document will be stored in DB or may by as a static file in AWS S3 so no heavy load will be inflicted to GitHub, but I like the idea to use Git and GitHub for CMS purposes.</p>
<p>I read the terms but found no explicit limitations of projects content type, it only should be kind and polite, but it looks like no one forces it to be a library or software or something like this.</p>
<p>So if, for example, I am a writer, can I have a GitHub project for my novel?</p>
<p>If I am tourist - can I have a plan of my future jorney, with plans, maps and so on?</p>
<p>Is it ok if such content will be machine-readable and will be managed by side app (editor)?</p>

**Comment by MChevy422** [comment]
<p>Hello <a class="mention" href="/yurigor">@yurigor</a>,</p>
<p>Welcome!</p>
<p>This post was moved to a different board that fits your topic of discussion a bit better. This means you’ll get better engagement on your post, and it keeps our Community organized so users can more easily find information.</p>
<p>As you’ll notice, your Topic is now in the ‘How to use Git and Github’ board. No action is needed on your part; you can continue the conversation as normal here.</p>
<p>Let me know if you have any other questions or if I can help with anything else.</p>
<p>Greatly appreciated,</p>
<p><strong>-MChevy422</strong></p>

**Comment by YuriGor** [comment]
<p>Thank you <a class="mention" href="/mchevy422">@mchevy422</a> for moving my question to the correct board, it’s my first post in the forum as you see, so I was disoriented a bit.</p>
<p>Well, may be you can help me with my question itself, do you know something about content type limitations in the terms of service?</p>
<p>May be a link to related article/chapter, which I missed?</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/yurigor">@yurigor</a>,</p>
<p>Thank you for reaching out! Here is a link to our <a href="https://help.github.com/articles/github-corporate-terms-of-service/" rel="nofollow noopener">terms of service</a> for your reference. In short yes, you might absolutely use GH for your projects regardless if they are or not code related. Maybe some of our user’s can share the unconventional ways they use GitHub here? <a href="https://www.infoworld.com/article/2886828/collaboration-software/github-for-the-rest-of-us.html" rel="nofollow noopener">Here is a compilation of some interesting ones from an external blog</a>.</p>
<p>Best,</p>
<p>Andrea</p>

**Comment by YuriGor** [comment]
<p>Thank you for reply, sounds like in general answer for my question is ‘yes’</p>

**Comment by BudiSurya** [comment]
sex vidwo
